### PR TITLE
Custom attributes replacement

### PR DESCRIPTION
**Pull request by RafaelGCruvinel**
# Enhancements to HTML-to-JSON Converter

This pull request introduces several enhancements to our HTML-to-JSON converter for the ChaiBuilder project:

Custom Attribute Replacement:

Implemented support for generic attribute replacement using the `data-chai-replace-*` prefix. This allows for dynamic substitution of HTML attributes with specified values, providing greater flexibility and customization.

Enhanced the attribute processing logic to better handle custom attributes and replace specified attributes based on the `data-chai-replace-*` prefix.

Example: `<div data-chai-replace-_name="CustomBlockName">Content</div>` will replace the _name attribute with "CustomBlockName".

### CODE REVIEW

File: src/core/import-html/html-to-json.ts [hunk 1]
```diff
@@ -111,6 +113,13 @@ const getAttrs = (node: Node) => {
     }
   });
 
+  forEach(attributes, ({ key, value }) => {
```
Comment (surajair) [inline src/core/import-html/html-to-json.ts:116]: @RafaelGCruvinel The attributes once replaced must be deleted so they dont add up in block json

File: src/core/import-html/html-to-json.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { STYLES_KEY } from "../constants/CONTROLS";
 import { omit } from "lodash";
 import { getVideoURLFromHTML, hasVideoEmbed } from "./import-video.ts";
 
+const DATA_CHAI_REPLACE_PREFIX = "data-chai-replace-";
```
Comment (surajair) [inline src/core/import-html/html-to-json.ts:10]: @RafaelGCruvinel I think we should use a smaller attr name for quickly adding attrs while developing html.

**Review by surajair — CHANGES_REQUESTED** [review]

File: src/core/import-html/html-to-json.ts (hunk 2 above)
Comment (RafaelGCruvinel) [reply src/core/import-html/html-to-json.ts:10]: yes.. the only point is to use "data-*".. do you have any recomendation? I think a little about it..

Comment (surajair) [reply src/core/import-html/html-to-json.ts:10]: i think simply `data-chai-{key}` should be good.

**Comment by surajair** [comment]
Closing this PR. this feature is added to import logic using `$name` attribute

**Pull request closed without merging**